code/modules/cm_marines/equipment/sentries.dm

```diff
@@ -284,9 +285,10 @@
 	spark_system.set_up(5, 0, src)
 	spark_system.attach(src)
 	cell = new /obj/item/cell/high(src)
+	camera_number = rand(1,1000)
```

Comment:
Eh there could be conflicts but not sure if doing something more complex would be worth it.

Reply:
I made `UNIQUEID` for the laser targets. Could be used for cameras as well.